File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/InitRange.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class InitRange{

	@SerializedName("start")
	private String start;

	@SerializedName("end")
	private String end;

	public void setStart(String start){
		this.start = start;
	}

	public String getStart(){
		return start;
	}

	public void setEnd(String end){
		this.end = end;
	}

	public String getEnd(){
		return end;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/InitRange.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class InitRange : Parcelable, Serializable {
    @SerializedName("start")
    var start: String? = null

    @SerializedName("end")
    var end: String? = null
    override fun toString(): String {
        return "InitRange{" +
                "start = '" + start + '\'' +
                ",end = '" + end + '\'' +
                "}"
    }
}